packages/paste-website/gatsby-config.js

```diff
@@ -42,12 +42,39 @@ module.exports = {
         path: `${__dirname}/src/pages/components`,
       },
     },
+    {
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        name: 'assets',
+        path: `${__dirname}/src/assets`,
+      },
+    },
+    `gatsby-transformer-sharp`,
+    `gatsby-plugin-sharp`,
+    {
+      resolve: `gatsby-transformer-remark`,
+      options: {
+        plugins: [{
+          resolve: `gatsby-remark-images`,
+          options: {
+            maxWidth: 1632,
+            linkImagesToOriginal: false,
+          },
+        }, ],
+      }
+    },
     {
       resolve: `gatsby-plugin-mdx`,
       options: {
         defaultLayouts: {
           default: require.resolve('./src/layouts/DefaultLayout.tsx'),
         },
+        gatsbyRemarkPlugins: [{
```

Comment:
Theres a bug in `gatsby-plugin-mdx` that doesn't allow `gatbsy-remark-images` to do its thing. Adding `gatsby-tranformer-remark` solves that issue.